src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/PostEntity.java

```diff
@@ -125,9 +126,13 @@ public String findAuthorId() {
   }
 
   public boolean hasEditedBySection() {
-    return post.findElement(By.cssSelector(".discussion-content + div"))
-        .getAttribute(CLASS_ATTRIBUTE)
-        .contains(POST_EDITED_BY_CLASS_NAME);
+    try {
+      return post.findElement(By.cssSelector(".post-edited-by-row"))
+          .getAttribute(CLASS_ATTRIBUTE)
+          .contains(POST_EDITED_BY_CLASS_NAME);
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)